index.js

```diff
@@ -21,11 +21,10 @@ const parseValue = (value, href) => {
 
 // Get `<a>` element as string
 function linkify(href, options = {}) {
-	// The regex URL mistakenly includes a dot at the end of the URL
-	let trailingDot = '';
-	if (href.endsWith('.')) {
+	// The URL regex mistakenly includes punctuation (a period or question mark) at the end of the URL
+	const punctuation = /[.?]$/.exec(href) ?? '';
```

Comment:
@fregante I guess it needs to be this actually

```suggestion
	const punctuation = /[.?]$/.exec(href)?.[0] ?? '';
```

Reply:
Oh TIL actually not necessary!

(because of the `.toString()` method on the `.exec()` result)

I think it's probably more expressive with the `?.[0]`

But if preferred, we can rely on the `.toString()` "magic" and I can revert this change

![Screenshot 2024-11-11 at 09 34 51](https://github.com/user-attachments/assets/ef934d32-04ee-4393-84b6-7c7204fa58f7)